src/__tests__/services/pokeService.test.js

```diff
@@ -0,0 +1,7 @@
+import { getPokemonData } from "../../services";
+import { mockPokemonData } from "../../mock/pokeData";
+
+it("Should get pokemon data with same object structure", async () => {
+  const data = await getPokemonData();
+  expect(data).toMatchObject(mockPokemonData);
```

Comment:
wow, now that you are using snapshots, play around with how styling make those snapshots change over time